packages/markup-frontstage/src/services/SavedView.ts

```diff
@@ -71,9 +80,72 @@ export const parseFromEmphasizeElements = (
 };
 
 /**
- * Creates saved view data
+ * Creates a drawing saved view.
+ * @param vp the name of the view port.
+ */
+const createDrawingSavedView = (vp: Viewport): SavedViewData => {
```

Comment:
this and createSheet... functions are very similar. Could they be combined and parameterized? Even if you leave the separate functions but they both call the single function with different parameters

Reply:
pulled the common into separate function, and parameterized the exposed function, used exposed function although it was not necessary for drawings, please let me know if you're good with it?